ddtrace/contrib/django/__init__.py

```diff
@@ -58,6 +58,15 @@
   be useful if you want to use the Django integration, but you want to trace only
   particular functions or views. If set to False, the request middleware will be
   disabled even if present.
+* ``AUTO_INSTRUMENT_TEMPLATE`` (default: ``{'EXCEPTION_MIDDLEWARE': True,
+  'CACHE': True, 'DATABASE': True, 'TEMPLATES': True}``): speficies which part of
+  the framework should be instrumented. This setting will be ignored if
+  ``AUTO_INSTRUMENT`` is set to ``False``. ``EXCEPTION_MIDDLEWARE`` controls
```

Comment:
we must be sure it's clear that if `AUTO_INSTRUMENT` is `False`, all the instrumentation is entirely disabled